client/src/components/Nav.js

```diff
@@ -130,9 +130,9 @@ const Nav = ({
             id="my-tasks-nav"
           >
             {`My ${pluralize(taskShortLabel)}`}
-            {notifications?.myTasksWithPendingAssessments?.length ? (
+            {notifications?.tasksWithPendingAssessments?.length ? (
```

Comment:
Is there a specific reason to rename this? having `my` at the start shows it is currentUser's tasks, not all tasks.

Reply:
They're notifications of any given position (the methods are generic, and not specifically linked to the current user's position; see https://github.com/NCI-Agency/anet/blob/9f03b8f6d659e9f36440025c3b7a20f46fa66ce6/client/src/notificationsUtils.js ). So `notificationUtils.js` should not return them as `my*`, since they could be from any position.

Reply:
To me, the notifications should be personal / aimed at the current user, the helper method that returns the pending stuff should be the generic one, like this (https://github.com/NCI-Agency/anet/blob/9f03b8f6d659e9f36440025c3b7a20f46fa66ce6/client/src/notificationsUtils.js#L31-L43).